Pipfile

```diff
@@ -4,6 +4,10 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-# Other than `pipenv` that is used to configure individual projects, this Pipfile should not have any other packages defined.
-# Individual projects should define their own packages/dependencies in repsective folders/Pipfiles.
-pipenv = "==2023.12.1"
+mkdocs = "==1.4.2"
+mkdocs-literate-nav = "==0.6.0"
+mkdocs-material = "==9.1.1"
+mkdocs-minify-plugin = "==0.6.2"
+mkdocs-pymdownx-material-extras = "==2.4.2"
+pipenv = "==2023.12.1" # Installed first, and used to install/run other dependencies
```

Comment:
nit: If the order matters, maybe it's worth separating it as the first and separate others with a newline?

Reply:
Ordering doesn't matter. But your suggestion improves readability as well. Will do!